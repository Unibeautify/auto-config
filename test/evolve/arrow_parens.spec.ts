import { testGenetic } from "../testGenetic";
import prettier from "@unibeautify/beautifier-prettier";
import jsBeautify from "@unibeautify/beautifier-js-beautify";

testGenetic("arrow_parens", {
  beautifiers: [prettier, jsBeautify],
  language: "JavaScript",
  originalText: `a => {};`,
  desiredText: `(a) => {};`,
});
