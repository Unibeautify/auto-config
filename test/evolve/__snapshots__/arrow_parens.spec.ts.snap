// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`arrow_parens: beautifiedText 1`] = `(a) => {};`;

exports[`arrow_parens: entity 1`] = `
Object {
  "options": Object {
    "arrow_parens": "always",
    "beautifiers": Array [
      "Prettier",
      "JS-Beautify",
    ],
  },
}
`;

exports[`arrow_parens: fitness 1`] = `1000`;
